src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerRecordListing.java

```diff
@@ -107,7 +109,9 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(CaseDetails<CaseDa
             DynamicList hearingVenueList = locationService.getHearingVenuesByRegion(regionId);
             caseData.getRecordListing().setHearingVenues(hearingVenueList);
 
-            String hearingVenueMessage = hearingVenueList.getListItems().isEmpty() ? "Unable to retrieve Hearing Venues data" : "";
+            String hearingVenueMessage = hearingVenueList == null || hearingVenueList.getListItems().isEmpty()
```

Comment:
Use CollectionUtils.isEmpty here?

Reply:
hearingVenueList would never be empty so only not null check would be sufficient.